Formula/ipopt.rb

```diff
@@ -0,0 +1,46 @@
+class Ipopt < Formula
+  desc "Large-scale nonlinear optimization package"
+  homepage "https://projects.coin-or.org/Ipopt"
+  url "https://www.coin-or.org/download/source/Ipopt/Ipopt-3.12.9.tgz"
+  sha256 "8ff3fe1a8560896fc5559839a87c2530cac4ed231b0806e487bfd3cf2d294ab8"
+  head "https://projects.coin-or.org/svn/Ipopt/trunk", :using => :svn
+
+  option "without-test", "Skip build-time tests (not recommended)"
+
+  depends_on "ampl-mp"
+  depends_on "gcc"
```

Comment:
Why do we need GCC?

Reply:
I added this because [`ipopt` had dependency of `:fortran`](https://github.com/Homebrew/homebrew-science/blob/19f75951641d3a5e70ea105f76a6a77bc0553d07/ipopt.rb#L21) by https://github.com/Homebrew/homebrew-science/commit/422a98e938358f2182b13120d973c25eddd12be1. But it looks like a transitive dependency from `mumps` so I believe we can remove.